ajax/libs/libphonenumber/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "libphonenumber",
+  "filename": "phonenumberutil.min.js",
+  "version": "7.7.4",
+  "description": "Google's common Java, C++ and JavaScript library for parsing, formatting, and validating international phone numbers.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/googlei18n/libphonenumber.git"
+  },
+  "keywords": [
+    "google",
+    "phonenumber"
+  ],
+  "author": "googlei18n <https://developers.google.com/international>",
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/googlei18n/libphonenumber.git",
+    "fileMap": [
+      {
+        "basePath": "javascript/i18n/phonenumbers",
+        "files": [
+          "asyoutypeformatter!(_test*)",
+          "metadata!(fortesting*|lite*)",
+          "phonemetadata.pb*",
+          "phonenumber.pb*",
+          "phonenumberutil!(_test*)"
```

Comment:
I think I can combined `phonemetadata.pb*` and `phonenumber.pb*` to `+(phonemetadata.pb*|phonenumber.pb*)`, and the others remain unchanged.
What about this?